src/client/src/containers/AzureSubscriptions/index.tsx

```diff
@@ -129,7 +129,7 @@ class AzureSubscriptions extends React.Component<Props, IState> {
               )}
               disabled={!isLoggedIn}
               handleDetailsClick={setDetailPage}
-              useNormalButtons={this.isSelectionCreated(option.internalName)}
+              useNormalButtons={false}
```

Comment:
lol what are *normal* buttons?

Reply:
Unhighlighted buttons, the dark buttons are "normal buttons"

Reply:
If they're always going to be blue from now on, does this prop need to be here?